Tests/UKHO.MaritimeSafetyInformation.Web.UnitTests/Services/NMDataServiceTest.cs

```diff
@@ -308,6 +308,36 @@ public void WhenDownloadFssFileAsyncThrowsException_ThenShouldExecuteCatch()
             Assert.IsTrue(result.IsFaulted);
         }
 
+        [Test]
+        public void WhenDownloadZipFssFileIsCalled_ThenShouldReturnByteArray()
```

Comment:
As per previous comment:
```suggestion
        public void WhenDownloadFSSZipFileAsyncIsCalled_ThenShouldReturnByteArray()
```

Reply:
Done